lib/functions/general/extensions.sh

```diff
@@ -541,6 +541,7 @@ function enable_all_extensions_builtin_and_user() {
 	# loop over the files found; remove the prefix
 	for extension_file in "${extension_list[@]}"; do
 		extension_file="${extension_file#${SRC}/}"
+		extension_file="${extension_file#${USERPATCHES_PATH}/}"
```

Comment:
I am not sure if this would actually make a difference. As USERPATCHES_PATH resolves to ${SRC}/userpatches, the line 543 would have already stripped out the ${SRC} part. So the complete USERPATCHES_PATH will never match returning the same output as line 543 as per my understanding

Reply:
Yeah, you are right.

Reply:
just a note here. This can be simplified further. Line 543 is redundant as well. The `extension_name="${extension_file##*/}"` line removes all the directory components anyways